app/programs/page.tsx

```diff
@@ -0,0 +1,10 @@
+import ProgramsPageClient from './page-client';
+
+export const metadata = {
+    description: 'Browse verified Solana programs and check their verification status',
+    title: 'Verified Programs | Solana Explorer',
+};
+
+export default function ProgramsPage() {
+    return <ProgramsPageClient />;
```

Comment:
Another improvement that might be helpful: collect issues at Sentry.
There is a helper to wrap up data for Sentry: https://github.com/notEduardo/explorer/blob/cd9b92093d32d3a0c161d41ccd79a0031ecebbb3/app/address/%5Baddress%5D/idl/page.tsx#L5